float32 panPos
float32 tiltPos
